homeassistant/components/sabnzbd/quality_scale.yaml

```
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup: done
```

```suggestion
  action-setup: todo
```
done in `async_setup_entry` not `async_setup` (even though they are deprecated)

---

Thx, fixed with https://github.com/home-assistant/core/pull/131656